tests/php/ORM/DataObjectDuplicationTest.php

```diff
@@ -92,8 +92,8 @@ public function testDuplicate()
         );
         // Ids of each record are the same (only mapping content is duplicated)
         $this->assertEquals(
-            $orig->caribou()->getIDList(),
-            $duplicate->caribou()->getIDList()
+            $orig->caribou()->sort(null)->column('ID'),
+            $duplicate->caribou()->sort(null)->column('ID')
```

Comment:
```suggestion
            $orig->caribou()->column('ID'),
            $duplicate->caribou()->column('ID')
```
This test is at risk of sporadically failing

Reply:
Reverted